morerss/jike.py

```diff
@@ -61,8 +56,9 @@ def post2rss(data_plan, post):
       topic = post['topic']
       description += "来自圈子：<a href='https://m.okjike.com/topics/%s' target='_blank'>%s</a>" % (topic['id'], topic['content'])
 
+  import re
   item = PyRSS2Gen.RSSItem(
-    title = title,
+    title = re.split('[,，\.。;；!！\?？~]|<br/>', content)[0],
```

Comment:
You need to use a raw string here (or escape `\`), or there will be `DeprecationWarning: invalid escape sequence \.`.

Reply:
![image](https://user-images.githubusercontent.com/40143136/132132252-f7aa9a53-e543-439b-b55e-c90433ef92e2.png)

Could see the warning with Python 3.6.8, 3.7.4 and 3.9.0.

Reply:
You need to enable warnings to see them, e.g.:
```sh
export PYTHONWARNINGS=default,ignore::ResourceWarning
```